packages/admin-color-picker/src/core/HexInput.tsx

```diff
@@ -31,13 +31,7 @@ const HexInput: React.FC<IComponentProps & ColorPickerProps> = ({ value, classes
     <div className={classes.inputInner}>
         <div className={classes.inputInnerLeftContent}>
             {!palette || (palette && picker) ? (
-                <EditableInput
-                    style={resetInputStyles}
-                    value={value}
-                    onChange={(colorState) => {
-                        onChange(colorState.hex);
-                    }}
-                />
+                <EditableInput style={resetInputStyles} value={value} onChange={onChange} />
```

Comment:
is that onChange really correct?

in the docs it says https://casesandberg.github.io/react-color/#api-onChange

Reply:
@nsams Yes, the first parameter is the input-value, not a color-object.